Cargo.toml

```diff
@@ -14,13 +14,19 @@ wasm-bindgen = "0.2.83"
 x25519-dalek = { version = "2.0.0", features = ["static_secrets"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-blake2 = "0.10.4"
+blake2 = "0.10.5"
 chacha20poly1305 = "0.10.1"
-sp-core = "6.0.0"
+sp-core = { version = "21.0.0", default-features = false, features = [
+  "full_crypto",
+  "serde",
```

Comment:
Out of curiosity, why do we need the `serde` feature here?

Reply:
`SignedMessage` (which implements serialize/deserialize) stores the message as `sp-core::Bytes`, which means we need this.  I don't know why we use `Bytes` rather than `Vec<u8>`